config/jobs/kubernetes-sigs/apiserver-network-proxy/apiserver-network-proxy-presubmits-0.32.yaml

```diff
@@ -0,0 +1,127 @@
+presubmits:
+  kubernetes-sigs/apiserver-network-proxy:
+  - name: pull-apiserver-network-proxy-test-0-32
+    cluster: eks-prow-build-cluster
+    always_run: true
+    branches:
+    - release-0.32
+    skip_report: false
+    decorate: true
+    path_alias: sigs.k8s.io/apiserver-network-proxy
+    spec:
+      containers:
+      - image: public.ecr.aws/docker/library/golang:1.23.0
```

Comment:
@liangyuanpeng - actually, do you mind making this 1.23.6?

That will unblock https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/715

Reply:
Please also make apiserver-network-proxy-presubmits-master.yaml here consistent with this new 0.32.yaml

Reply:
Updated